### DISCUSSION TITLE
cobbler3.2.2 don't work

### DISCUSSION THREAD

**Discussion by dovevastai** [cobbler/cobbler#2892] [Version specific questions]
I meet a cobbler issue, info below:

- OS: Centos8.3
- Cobbler: 3.2.2

I build it refer to <https://cobbler.github.io/> but it doesn't work. I don't find some question in `/var/log/cobbler/cobbler.log`.

**Comment by SchoolGuy** [comment]
So what is not working? I don't see an issue described in your post sadly.
